nengo_gui/components/pointer.py

```diff
@@ -70,8 +70,18 @@ def javascript(self):
         return 'new Nengo.Pointer(main, sim, %s);' % json
 
     def message(self, msg):
-        if len(msg) == 0:
+        if msg == ':empty:':
             self.override_target = None
+        elif msg[0:12] == ':check only:':
+            if len(msg) == 12:
+                self.data.append("good_pointer")
+            else:                
+                vocab = copy.deepcopy(self.vocab_out)
+                try:
+                    vocab.parse(msg[12:])
+                    self.data.append("good_pointer")
+                except:
```

Comment:
This bare exception doesn't feel right to me. Then again, should this wait to be resolved until we have custom Nengo errors?